src/Stripe.net.Tests/Stripe.net.Tests.csproj

```diff
@@ -2,11 +2,12 @@
 
   <PropertyGroup>
     <VersionPrefix>0.0.0</VersionPrefix>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
     <AssemblyName>Stripe.net.Tests</AssemblyName>
     <PackageId>Stripe.net.Tests</PackageId>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <RuntimeIdentifiers>win</RuntimeIdentifiers>
+    <NetStandardImplicitPackageVersion>1.2</NetStandardImplicitPackageVersion>
```

Comment:
Same answer as above - I bring this up only because I'm assuming you're thinking stripe.net should target the same standard.